### DISCUSSION TITLE
How to install stackstorm to mac os m1 system?

### DISCUSSION THREAD

**Discussion by Abhinav-Mani** [StackStorm/st2#5863] [Q&A]
Need help installing stack storm on my mac m1 syatem

**Comment by ohcnkk** [comment]
You can use stackstorm docker compose. It’s easy to setup and interact with st2 client too. The repo is https://github.com/StackStorm/st2-docker

**Comment by dumbworker** [comment]
You might have to add `--platform=linux/amd64` in the dockerfile
